datafusion/common/src/config.rs

```diff
@@ -1628,38 +1616,15 @@ mod tests {
         );
     }
 
-    #[test]
-    fn parquet_table_options() {
-        let mut table_config = TableOptions::new();
-        table_config
-            .set("parquet.bloom_filter_enabled::col1", "true")
-            .unwrap();
-        assert_eq!(
-            table_config.parquet.column_specific_options["col1"].bloom_filter_enabled,
-            Some(true)
-        );
-    }
-
```

Comment:
It is not obvious to me why this is removed and not its CSV counterpart. It seems this could be adapted like the CSV test too, as in:
```rust
    #[cfg(feature = "parquet")]
    #[test]
    fn parquet_table_options() {
        let mut table_config = TableOptions::new();
        table_config.set_file_format(FileType::PARQUET);
        table_config
            .set("format.bloom_filter_enabled::col1", "true")
            .unwrap();
        assert_eq!(
            table_config.parquet.column_specific_options["col1"].bloom_filter_enabled,
            Some(true)
        );
    }
```